PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.java

```diff
@@ -137,10 +160,34 @@ protected void textChanged(DocumentEvent e) {
         getOKAction().setEnabled(false);
     }
 
+    protected void printLogLine(@NotNull ConsoleViewContentType logLevel, @NotNull String log) {
+        consoleViewPanel.print(LocalDateTime.now().toString() + " " + logLevel.toString().toUpperCase() + " " + log + "\n", logLevel);
```

Comment:
Could you use `jodatime` to replace `LocalDateTime`? the jodatime can provide time zone with date.

Reply:
And could you combine this function with the one of provision dialog?

Reply:
Replaced with `DateTime` in joda time. E.g. `2019-01-15T14:57:53.587+08:00`

---
In reply to: [247745657](https://github.com/Microsoft/azure-tools-for-java/pull/2663#discussion_r247745657) [](ancestors = 247745657)